18 gay frogs from chemicals

EbolaSpreadn247 opened the pull request:
i wanna add a tid bit to this

MeatyLearner commented on ewspooky.py:
```diff
@@ -285,7 +303,7 @@ async def haunt(cmd):
 
 	# Send the response to the player.
-	resp_cont.add_channel_response(cmd.message.channel.name, response)
+	resp_cont.add_channel_response(cmd.message.channel.name, response+channel_response)
```
`response` is the message the ghost sees. You'll want to append `channel_response` to `haunt_message` instead.

MeatyLearner commented on ewspooky.py:
```diff
@@ -225,7 +225,25 @@ async def haunt(cmd):
+				#channel item
+                                channel_response = ""
+                                if cmd.tokens_count > 2:
+                                        item_search = ewutils.flattenTokenListToString(cmd.tokens[2:])
```
the item the user wants to use is in the second token, so you would use tokens[1]. [2:] reads every argument after the second token, and you gotta account for the haunt text players can also add.

MeatyLearner replied:
speaking of the haunt text actually, you also gotta change the way these items are handled to support it. the item shouldn't be included in the message

MeatyLearner commented on ewspooky.py:
```diff
@@ -225,7 +225,25 @@ async def haunt(cmd):
+                                        item_search = ewutils.flattenTokenListToString(cmd.tokens[2:])
+                                        item_sought = ewitem.find_item(item_search = item_search, id_user = cmd.message.author.id, id_server = cmd.guild.id if cmd.guild is not None else None, item_type_filter = ewcfg.it_item)
+				        if item_sought != None:
+                                                  channel_power = ewcfg.item_hauntpower(item_sought.item_name)
```
`item_hauntpower` isn't a function so this will raise an error. do `ewcfg.item_hauntpower.get(item_sought.item_name)`

MeatyLearner commented on ewspooky.py:
```diff
@@ -225,7 +225,25 @@ async def haunt(cmd):
+                                                         await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, response)) 
+                                                  
+						  
+                                                  haunt_power_multiplier *= channel_power        
```
this will also raise an error if the item sought is not in the `item_hauntpower` list

MeatyLearner reviewed (commented):
besides the other stuff i listed, there also seem to be indentation issues

EbolaSpreadn247 reviewed (commented):
uh. i forgot what i was going to do extra. is this code right?

Zugerujk committed:
Fixes for things

Zugerujk committed:
Did you know Alex Jones was right?

They literally did put chemicals in the water that did turn frogs gay. It's hilarious.

Zugerujk committed:
Anoint fix, autobalance change

Changes anoint so that it shouldn't have any issues anymore with flavor and failing. Makes autobalance level 10 by default again.
